core/src/main/java/hudson/console/AnnotatedLargeText.java

```diff
@@ -231,17 +235,29 @@ public long writeHtmlTo(long start, Writer w) throws IOException {
         ConsoleAnnotationOutputStream<T> caw = new ConsoleAnnotationOutputStream<>(
                 w, createAnnotator(Stapler.getCurrentRequest2()), context, charset);
         long r = super.writeLogTo(start, caw);
+        long bytesRead = 0;
+
+        try (Writer writer = w) {
+             bytesRead = super.writeLogTo(start, writer);
+            writer.flush();
 
+        }
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         Cipher sym = PASSING_ANNOTATOR.encrypt();
-        ObjectOutputStream oos = AnonymousClassWarnings.checkingObjectOutputStream(new GZIPOutputStream(new CipherOutputStream(baos, sym)));
-        oos.writeLong(System.currentTimeMillis()); // send timestamp to prevent a replay attack
+        ObjectOutputStream oos = AnonymousClassWarnings.checkingObjectOutputStream(
+                new GZIPOutputStream(new CipherOutputStream(baos, sym))
+        );
+
+        oos.writeLong(System.currentTimeMillis());
```

Comment:
Seems to just be formatting changes (and deleting a comment)? Please try to keep diffs minimal.